packages/flutter_driver/lib/src/extension/internal_wait.dart

```diff
@@ -0,0 +1,174 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/widgets.dart';
+
+import '../common/wait.dart';
+
+/// Base class for a condition that can be waited upon.
```

Comment:
This should document that if you subclass this you probably also want to implement a [SerializableWaitCondition] and say something about the relationship between the two.

Reply:
Done. PTAL.